tests/integration/targets/vmware_ops_provision_virtual_esxi_test/vars/main.yml

```diff
@@ -12,7 +12,7 @@ provision_virtual_esxi_datacenter: "{{ vcenter_datacenter }}"
 provision_virtual_esxi_datastore_iso_path: "{{ custom_esxi_8_iso_path }}"
 resource_pool_name: ci-resource-pool-test
```

Comment:
I guess `tiny_prefix` should be added to `resource_pool_name` as well

Reply:
done